test/integration/test.ads.js

```diff
@@ -214,7 +154,7 @@ QUnit.test('changing the src triggers "contentupdate"', function(assert) {
   this.player.on('contentupdate', spy);
 
   // set src and trigger synthetic 'loadstart'
-  this.player.src('http://media.w3.org/2010/05/sintel/trailer.mp4');
+  this.player.currentSrc = () => 'AH HA!!! I AM NOT A REAL SOURCE';
```

Comment:
@incompl do we want to change this back?

Reply:
It's kind of nice that this demonstrates that a source isn't really being loaded, but we could also make the string less annoying.